### ISSUE TITLE
[BUG] Raid Notifications Filtered out "Egg hatch in -1501022147.58 seconds"

### ISSUE THREAD

**Issue by Toetje585** [kvangent/PokeAlarm#411]
!--- [BUGUSER SPECIFIC] Use this template if PokeAlarm is behaving in an incorrect or unexpected way. ---
## Problem Description

A few hours PokeAlarm was running just fine until recently, all my filters are getting ignored:

"Ignored. Egg hatch in -1501022677.37 seconds"

I rebooted the hole machine recreated the docker images no idea what is going on here. Restarted the docker instances a few times.

## Your Environment
!--- Please post as many details  --

Synology Nas with Docker
Python Version 2.7.12

docker run -d --name pokepokemaastricht_gymlvl5 --net=pogonw \
    pokealarmgymslvl5 \
    -k *****************************

Webhooks configured in config.ini (Rocketmap)

webhook: [http://pokepokemaastricht_gym:4000,http://pokepokemaastricht_gymlvl5:4001]

Did the issue persist after restarting the script?

Yes

## Console Output
!--- Please post any relevant output provided from the PokeAlarm when you ran it --
```
	
2017-07-25 22:35:47,582 [      manager_0][   Manager][    INFO] Egg OTM0YzY5ZjU3NGQ0NDUyZWIwYzc2ZjFlNDZhNDYwNjIuMTI= ignored. Egg hatch in -1501022147.58 seconds | stderr


2017-07-25 22:37:51,888 [      manager_0][   Manager][    INFO] Egg OTBmMzgyMGQyOTYxNGUyOGIzNTM2YWVlYzQ5ZjNkOGQuMTY= ignored. Egg hatch in -1501022271.89 seconds | stderr
-- | --


```

## Your Alarms Config
!--- Please post the config for your alarms in the space below --
!--- Remove all person data such as API_KEYs or CHAT_IDs --
```json
    
[
  {
    "active": "True",
    "type": "discord",
    "webhook_url": "https://discordapp.com/api/webhooks/339370400825737216/MeKlnC47SUUJWbveEjw0ds-*****************"
  },

```

## Troubleshooting Checklist
[X ] Most recent version of PokemonGo-Map
[X] Most recent version of PokeAlarm
[ ] Checked the Wiki page for service in question

**Comment by kvangent** [comment]
My understanding is that Raids currently don't send eggs at the moment. Is this still correct?

**Comment by Toetje585** [comment]
I Only got notified about a egg hatching once in Discord but it's time was 0.00 left. But according to the community there aren't any raid that send eggs indeed. However I just started PokeAlarm again and all my channels are getting notified again. So perhaps something else was going on aswell.

In my debug window I now see this:

**`Raid on Magmar ignored: no filters are set`**

I was missing this yesterday for a few hours, While I know it ignores them due my filters for some time yesterday it wasn't detecting any raid. Then a hour or two later the hashing servers wen't down and this morning when I started all scripts again everything is back to normal.

We keep monitoring.. :)
